exercises3/removeDuplicates.js

```diff
@@ -0,0 +1,8 @@
+/*
+Implement a JavaScript function to remove 
+duplicate elements from an array.
+*/
+
+const removeDuplicates = array => {
```

Comment:
Remove comments!
 Can you use implicit returns as this function has a single expression?